### PR TITLE
Release v1.110.0

### PR DESCRIPTION
**Pull request by confused-Techie**
As title says.

Relates to #765

### CODE REVIEW

**Comment by DeeDeeG** [comment]
Dunno how big a deal this test failure is, I'm betting not very?

> ```
> find-install-method main
>   it Returns developer instance if applicable
>     Expected 'Spec Mode' to be 'Developer Instance'.
>       at jasmine.Spec.<anonymous> (/home/runner/work/pulsar/pulsar/node_modules/pulsar-updater/spec/find-install-method-spec.js:14:34)
> ```

https://github.com/pulsar-edit/pulsar/actions/runs/6527478245/job/17722644597?pr=766#step:10:42

This happened last Regular release PR as well: https://github.com/pulsar-edit/pulsar/actions/runs/6205040177/job/16847918274

I think Regular version numbers are just an edge case that the test doesn't handle right at the moment.

File: package.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "name": "pulsar",
   "author": "Pulsar-Edit <admin@pulsar-edit.dev>",
   "productName": "Pulsar",
-  "version": "1.109.0-dev",
+  "version": "1.110.0",
```
Comment (DeeDeeG) [inline package.json:5]:
```suggestion
  "version": "1.110.0-dev",
```

Doing this as a code suggestion is probably the easiest way to get the `-dev` added back to the version. Heh.

Any way we do that, I would Approve, not that we really need Approvals on the version bump PRs since we're generally multiple people on-deck in real-time anyhow for these releases...
